tests/test_conda_forge_activity.py

```diff
@@ -27,6 +28,17 @@ def test_feedstock_url(name, proto, exp):
     assert exp == obs
 
 
+conver_feedstock_url_tests = []
+for name in ['git@github.com:conda-forge/my-feedstock.git',
+             'http://github.com/conda-forge/my-feedstock.git',
+             'https://github.com/conda-forge/my-feedstock.git']:
+    for proto, exp in zip(['ssh', 'http', 'https'],
+                          ['git@github.com:conda-forge/my-feedstock.git',
+                           'http://github.com/conda-forge/my-feedstock.git',
+                           'https://github.com/conda-forge/my-feedstock.git']):
+        conver_feedstock_url_tests.append((name, proto, exp))
+
```

Comment:
What is the purpose of the above if it isn't used ?

Reply:
Sorry about that